### PR TITLE
Move things at global namespace to proper location

### PR DESCRIPTION
**Pull request by Fadis**
utilities at global namespace -> fuzzuf::utils
executors at global namespace -> fuzzuf::executor
ExecInput at global namespace -> fuzzuf::exec_input
feedbacks at global namespace -> fuzzuf::feedback
python binding at global namespace -> fuzzuf::bindings::python
cli implementation at global namespace -> fuzzuf::cli
hierarflow at global namespace -> fuzzuf::hierarflow
Fuzzer -> fuzzuf::fuzzer::Fuzzer
mutator at global namespace -> fuzzuf::mutator

## Type of PR

- [ ] Changes related to the roadmap (e.g., TODO.md) (type: A) -> Create an issue corresponding to the PR in advance, and refer to this PR in the issue.
- Changes that are not related to the roadmap
    - [ ] Change with multiple possible solutions to the issue (type: B-1) -> Create an issue corresponding to the PR in advance and refer to this PR in the issue.
    - [x] Change with a single solution (type: B-2) -> There is no need to create an issue corresponding to the PR in advance. Please discuss it in this PR.

## Importance of PR

- Importance of the issue
    - [ ] Large (based on several days to weeks of discussion and verification, e.g., this issue is a blocking issue for other issues on the roadmap, etc.)
    - [ ] Medium (based on a few hours to a day of discussion and verification, e.g., this issue is a blocking issue for another minor issue)
    - [x] Small (apparent changes such as build error)
- Complexity of the solution (code, tests, etc.)
    - [ ] Large (requires several days to several weeks of review)
    - [ ] Medium (requires several hours to a day of review)
    - [x] Small (trivial changes, such as build error)

## Concerns (Optional)

- [ ] Performance
- [ ] Source Code Quality

---

> The PR author should fill in the following checklist when submitting this PR.

#### Optional Entries
- [ ] If this PR is a PR type A/B-1, there is a cross-link between this PR and the related issue.

#### Mandatory Entries
- [ ] The PR title is a summary of the changes.
- [ ] Completed each required field of the PR.

---
> The PR author should fill out the following checklist in the comments to confirm that this PR is ready to be merged

- [ ] CI is green or confirmed test run results.
- [ ] All change suggestions from reviewers have been resolved (fixed or foregone).

---
> The maintainer of this repository will set up a reviewer for each PR.
> PR reviewers should review this PR in terms of the checklist below before moving on to a detailed code review. Please comment on their initial response by filling in the checklist below.

#### Optional Entries
- [ ] The reviewer assigned more reviewers if needed.
- [ ] The reviewer noted that it is necessary to break out some of the changes in this PR into other PRs if needed.
- [ ] The reviewer noted that the initial response is insufficient if needed.

#### Mandatory Entries
- [ ] The title of this PR summarizes the changes made by this PR properly.
- [ ] The target branch of this PR is as intended.
- [ ] The reviewer understands the issues in this PR.
- [ ] The reviewer plans to review with an appropriate workload based on the importance of this PR.

---
> When the PR reviewer concludes that this PR is ready to be merged, please fill in the checklist below by posting it in the comment. If there is more than one reviewer, please do this on your own.

#### Optional Entries
- [ ] The reviewer noted that if you believe that new tests are needed to evaluate this PR, they have been noted.
- [ ] If minor refactorings are not mentioned in the PR, I understand the intent.
- [ ] If this PR is a PR type A/B-1, we have agreed on this PR's design, direction, and granularity in the related issue.

#### Mandatory Entries
- [ ] The reviewer understands how this PR addresses the issue and the specific changes.
- [ ] This PR uses the best possible issue resolution that the reviewer can think of.
- [ ] This PR is ready to be merged.

### CODE REVIEW

**Comment by Fadis** [comment]
run tests

File: include/fuzzuf/mutator/templates/mutator.hpp [hunk 1]
```diff
@@ -22,275 +22,270 @@
...
-  -2147483648LL, /* Overflow signed 32-bit when decremented */ \
-  -100663046,    /* Large negative number (endian-agnostic) */ \
-  -32769,        /* Overflow signed 16-bit                  */ \
-   32768,        /* Overflow signed 16-bit                  */ \
-   65535,        /* Overflow unsig 16-bit when incremented  */ \
-   65536,        /* Overflow unsig 16 bit                   */ \
-   100663045,    /* Large positive number (endian-agnostic) */ \
-   2147483647    /* Overflow signed 32-bit when incremented */
+#define INTERESTING_8                                    \
+  -128,    /* Overflow signed 8-bit when decremented  */ \
```
Comment (retrage) [inline include/fuzzuf/mutator/templates/mutator.hpp:30]: 先頭だけ揃っていないのが気になる

File: include/fuzzuf/mutator/templates/mutator.hpp [hunk 2]
```diff
@@ -22,275 +22,270 @@
...
+      0,   /*                                         */ \
+      1,   /*                                         */ \
+      16,  /* One-off with common buffer size         */ \
+      32,  /* One-off with common buffer size         */ \
+      64,  /* One-off with common buffer size         */ \
+      100, /* One-off with common buffer size         */ \
+      127  /* Overflow signed 8-bit when incremented  */
+
+#define INTERESTING_16                                    \
+  -32768,   /* Overflow signed 16-bit when decremented */ \
```
Comment (retrage) [inline include/fuzzuf/mutator/templates/mutator.hpp:41]: 先頭だけ揃っていないのが気になる

File: include/fuzzuf/mutator/templates/mutator.hpp [hunk 3]
```diff
@@ -22,275 +22,270 @@
...
+      255,  /* Overflow unsig 8-bit when incremented   */ \
+      256,  /* Overflow unsig 8-bit                    */ \
+      512,  /* One-off with common buffer size         */ \
+      1000, /* One-off with common buffer size         */ \
+      1024, /* One-off with common buffer size         */ \
+      4096, /* One-off with common buffer size         */ \
+      32767 /* Overflow signed 16-bit when incremented */
+
+#define INTERESTING_32                                          \
+  -2147483648LL,  /* Overflow signed 32-bit when decremented */ \
```
Comment (retrage) [inline include/fuzzuf/mutator/templates/mutator.hpp:53]: 先頭だけ揃っていないのが気になる

File: include/fuzzuf/algorithms/libfuzzer/create.hpp [hunk 4]
```diff
@@ -294,45 +296,37 @@ auto createRunone(const FuzzerCreateInfo &create_info,
...
-    else {
+          hf::CreateNode<StaticAssign<F, decltype(Ord::stuck_count)>>(0u);
+
+      symcc << (if_no_new_coverage
+                    << (increment_stuck_count ||
+                        threshold
+                            << (reset_stuck_count1 ||
+                                createSymCC<F, Ord>(create_info, false, true,
+                                                    false, false, sink))) ||
+                if_new_coverage << (reset_stuck_count2));
```
Comment (retrage) [inline include/fuzzuf/algorithms/libfuzzer/create.hpp:323]: Hierarflowの部分だと思うんですが、clang-format適用されているのは意図通りでしょうか？

File: algorithms/ijon/ijon_hierarflow_routines.cpp [hunk 5]
```diff
@@ -97,7 +97,6 @@ using IJONUpdCalleeRef = afl::routine::update::AFLUpdCalleeRef;
 IJONUpdate::IJONUpdate(IJONState &state, std::size_t offset_)
     : state(state), offset(offset_) {}
 
-UpdateMax::UpdateMax(IJONState& state) : state(state) {}
```
Comment (retrage) [inline algorithms/ijon/ijon_hierarflow_routines.cpp]: このメソッドが消されているのは意図通りですか？

File: executor/linux_fork_server_executor.cpp [hunk 6]
```diff
@@ -94,7 +94,6 @@ LinuxForkServerExecutor::LinuxForkServerExecutor(
         fuzzuf::utils::OpenFile(path, O_RDWR | O_CREAT | O_CLOEXEC, 0600);
   } else {
     input_fd = fuzzuf::utils::OpenFile(args.path_to_write_input.string(),
-                                       O_RDWR | O_CREAT | O_CLOEXEC, 0600);
```
Comment (retrage) [inline executor/linux_fork_server_executor.cpp]: 9f7a8cdf2751627e240362bf17906d38bc49be01 で再度追加されている。このコミットで消す必要はないはず。

**Review by retrage — COMMENTED** [review]
修正ありがとうございます。全体としてよさそうです。一応、漏れがないかDoxygen生成のHTMLで確認しました。以下のソースコードはnamespaceが入っていないようです。意図しているものか確認をお願いします。
include/fuzzuf/algorithms/afl/count_classes.hpp:CountClasses
include/fuzzuf_cc/execute_put_api_response.h:ExecutePUTAPIResponse
include/fuzzuf/executor/executor.hpp:Executor
include/fuzzuf/utils/common.hpp:InvalidOption
include/fuzzuf/utils/common.hpp:FileError
include/fuzzuf/cli/fuzzer_args.hpp:FuzzerArgs
include/fuzzuf/hierarflow/parent_traversable.hpp:ParentTraversable

File: algorithms/ijon/ijon_hierarflow_routines.cpp (hunk 5 above)
Comment (Fadis) [reply algorithms/ijon/ijon_hierarflow_routines.cpp]: Expected changes. The class is not used in final IJON implementation,  therefore the class is not needed.

File: executor/linux_fork_server_executor.cpp (hunk 6 above)
Comment (Fadis) [reply executor/linux_fork_server_executor.cpp]: fixed

File: include/fuzzuf/algorithms/libfuzzer/create.hpp (hunk 4 above)
Comment (Fadis) [reply include/fuzzuf/algorithms/libfuzzer/create.hpp:323]: expected changes

**Comment by Fadis** [comment]
Pointed things still in the global namespace accepting common.hpp are fixed. Since changes on common.hpp stuff will require large changes, those issues will be tackled tomorrow.

File: include/fuzzuf/mutator/templates/mutator.hpp (hunk 1 above)
Comment (Fadis) [reply include/fuzzuf/mutator/templates/mutator.hpp:30]: Impossible to fix. automatically applied clang-format reverts changes.

File: include/fuzzuf/mutator/templates/mutator.hpp (hunk 2 above)
Comment (Fadis) [reply include/fuzzuf/mutator/templates/mutator.hpp:41]: Impossible to fix. automatically applied clang-format reverts changes.

File: include/fuzzuf/mutator/templates/mutator.hpp (hunk 3 above)
Comment (Fadis) [reply include/fuzzuf/mutator/templates/mutator.hpp:53]: Impossible to fix. automatically applied clang-format reverts changes.

**Pull request merged**